src/Ouroboros/Network/Protocol.hs

```diff
@@ -1,5 +1,9 @@
-{-# LANGUAGE RecordWildCards     #-}
-{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE BangPatterns               #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE NamedFieldPuns             #-}
+{-# LANGUAGE RecordWildCards            #-}
+{-# LANGUAGE ScopedTypeVariables        #-}
+{-# LANGUAGE StandaloneDeriving         #-}
```

Comment:
Are these extra language pragmas right? I don't immediately see them being used.

Reply:
All changes to this whole module seem to be only whitespace. Can we avoid mixing unnecessary whitespace changes like this into the same PR. It makes it hard to see what we're doing.

We're not enforcing a silly layout rule with a tool are we?

Reply:
I haven't used any tool, I just didn't removed everything I should.